### ISSUE TITLE
[administrative] store stable code in a release branch and add CI tools

### ISSUE THREAD

**Issue by kirkscheper** [inilabs/caer#12]
I am a new user to cAER and a couple of days ago I cloned your repo but it failed to compile. It ended up that a recent change caused a build error. I had to edit the code to get it to build. Apparently this error was detected by someone from the developer team as it was later fixed.

I use a few other open source projects where regular releases are branched of into a branch which no longer receives updates. This approach also helps to set goals for what you need to do before the next release. Before a new release is branched off you must pass certain tests (e.g. compiling for all camera targets, operational data input from each target, operational tcp/udp links ect.).

Additionally, you may consider adding a Continuous Integration tool to ensure that pull requests do not unintentionally break the building process of any target. This tool can be set up to build cAER for each target camera type and lets you know if one of them fails to build. This is a great tool to test new code for simple bugs.

An example of this can be found here: https://github.com/paparazzi/paparazzi

**Comment by llongi** [comment]
I agree and we do have tags in place for that. I don't think a completely separate branch is needed for releases, tags are there for exactly that purpose. Right now we have a lot of work being done, and when 1.0.0 will be released, there will of course be the proper tagging.
Regarding CI, I'll look into it, the idea is very good.
Keeping this bug open until I've explored that.
